packages/server/src/sdk/app/views/index.ts

```diff
@@ -96,6 +96,13 @@ async function guardCalculationViewSchema(
       continue
     }
 
+    if (!schema.field) {
+      throw new HTTPError(
+        `Calculation field "${name}" is missing a "field" property`,
+        400
+      )
+    }
```

Comment:
This a separate problem I ran into accidentally. It would error previously, but the message was unhelpful. This just improves the error message.